src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/ItemsSearch.js

```diff
@@ -53,15 +55,16 @@ export default class ItemsSearch extends Component {
           type: recordToPidType(hits[0]),
           value: hits[0].metadata.pid,
         };
-        checkoutItem(documentPid, itemPid, patronDetails.user_pid, true);
+        await checkoutItem(documentPid, itemPid, patronDetails.user_pid, true);
```

Comment:
can you double check that there is a try/catch for these 2 async/await, just to be sure we catch the error?